cloudbuild.yaml

```diff
@@ -5,7 +5,7 @@ options:
   substitution_option: ALLOW_LOOSE
   machineType: 'E2_HIGHCPU_8'
 steps:
-  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud@sha256:4e830b673791d5595719bc6c4ca62dce3746b4e20d749e45004254bc6ef0a140' # v20250116-2a05ea7e3d
+  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud@sha256:fe4041d57c711070436d3c5b7702534b7158f464e9772aa9e333a5e0e24fd854' # v20250513-9264efb079
```

Comment:
I cannot pull this image, can you?

```
docker pull gcr.io/k8s-staging-test-infra/gcb-docker-gcloud@sha256:fe4041d57c711070436d3c5b7702534b7158f464e9772aa9e333a5e0e24fd854
Error response from daemon: manifest for gcr.io/k8s-staging-test-infra/gcb-docker-gcloud@sha256:fe4041d57c711070436d3c5b7702534b7158f464e9772aa9e333a5e0e24fd854 not found: manifest unknown: Requested entity was not found.
```

Reply:
I think this sha works `sha256:63840f133e0dfeea0af9ef391210da7fab9d2676172e2967fccab0cd6110c4e7`

Reply:
Thx! I fixed it.